content/en/database_monitoring/setup_oracle/_index.md

```diff
@@ -9,69 +9,24 @@ disable_sidebar: true
 <div class="alert alert-warning">Database Monitoring is not supported for this site.</div>
 {{< /site-region >}}
 
-## Supported Oracle versions, features, and architectures
-
-- **Versions**: 11.2, 12c, 18c, 19c, and 21c
-- **Deployment configurations**: Self-managed, RDS, RAC, Exadata, Autonomous Database
-- **Architecture**: Multi-tenant, non-CDB, RDS single-tenant
-
-## Overview
-
-### Prerequisites
-
-To configure Database Monitoring for Oracle, the following prerequisites must be met:
-
-1. An Agent version that supports Oracle monitoring features must be installed (`7.50.1` or greater).
-    - [Install the Agent](#install-the-agent)
-    - [Upgrade an existing Agent installation](#upgrade-an-existing-agent-installation)
-2. The Oracle integration must be installed.
-    - [Install the Oracle integration](#install-the-oracle-integration)
-    - [Verify an existing Oracle integration installation](#verify-your-existing-oracle-integration)
-
-### Setup
-If the above prerequisites are met, follow the setup instructions for your hosting type:
+## Supported Oracle versions
+
+|            | Self-Hosted | RDS       | RAC       | Exadata   | Autonomous Database | Automatic Storage Management |
+|------------|-------------|-----------|-----------|-----------|---------------------|------------------------------|
+| Oracle 11g | {{< X >}}   | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}}           | {{< X >}}                    |
+| Oracle 12c | {{< X >}}   | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}}           | {{< X >}}                    |
+| Oracle 18c | {{< X >}}   | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}}           | {{< X >}}                    |
+| Oracle 19c | {{< X >}}   | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}}           | {{< X >}}                    |
+| Oracle 21c | {{< X >}}   | {{< X >}} | {{< X >}} | {{< X >}} | {{< X >}}           | {{< X >}}                    |
+
+## Supported architectures
+- multi-tenant
```

Comment:
lmk your thoughts here, got some feedback from sales that they don't make the connection that multi-tenant == CDB

we've been using these terms interchangeably but maybe it makes sense to write 

- `multi-tenant (i.e. CDB/PDB)`
- `non-CDB`

Reply:
`multi-tenant` and `CDB` are used interchangeably for this type of architecture, but if the experience shows that some customers and prospects are finding it ambiguous, it would be better to refer to it as `multi-tenant (i.e. CDB/PDB)`